Testing/SystemTests/tests/analysis/ISISPowderDiffractionPolarisTest.py

```diff
@@ -0,0 +1,363 @@
+        self.assertEquals(4413, data8.blocksize())
+        self.assertAlmostEqual(data7.readY(0)[6], data8.readY(0)[6], places=DIFF_PLACES)
+
+        self.assertEquals(1, data9.getNumberHistograms())
+        self.assertEquals(6107, data9.blocksize())
+        self.assertAlmostEqual(data9.readE(0)[3369], data10.readE(0)[3369], places=DIFF_PLACES)
+
+        self.assertEquals(1, data10.getNumberHistograms())
+        self.assertEquals(6107, data10.blocksize())
+        self.assertAlmostEqual(299.75529, data10.readX(0)[10], places=DIFF_PLACES)
+
+    def test_upstripped_files(self):
+        diff_places = 3
+
+        nxsfile = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                             "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped.nxs")
+
+        datfile1 = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                              "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped-0.dat")
+        datfile2 = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                              "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped-1.dat")
+        datfile3 = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                              "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped-2.dat")
+        datfile4 = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                              "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped-3.dat")
+        datfile5 = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                              "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped-4.dat")
+        datfile6 = (DIRS[0] + "POLARIS/test/Cycle_15_2/Calibration/"
+                              "POL_2015_2_5mm_vrod_78338_152_calfile_new_unstripped-5.dat")
+
+        nxsdata = LoadNexusProcessed(Filename=nxsfile, OutputWorkspace="nxs_workspace")
+        data0 = LoadAscii(Filename=datfile1, OutputWorkspace="dat_workspace0")
+        data1 = LoadAscii(Filename=datfile2, OutputWorkspace="dat_workspace1")
+        data2 = LoadAscii(Filename=datfile3, OutputWorkspace="dat_workspace2")
+        data3 = LoadAscii(Filename=datfile4, OutputWorkspace="dat_workspace3")
+        data4 = LoadAscii(Filename=datfile5, OutputWorkspace="dat_workspace4")
+        data5 = LoadAscii(Filename=datfile6, OutputWorkspace="dat_workspace5")
+
+        self.assertTrue(isinstance(nxsdata, MatrixWorkspace))
+        self.assertEquals(6, nxsdata.getNumberHistograms())
+        self.assertEquals(7793, nxsdata.blocksize())
+
+        self.assertAlmostEqual(nxsdata.readY(0)[0], data0.readY(0)[0], places=diff_places)
+        self.assertAlmostEqual(nxsdata.readY(0)[4500], data0.readY(0)[4500], places=diff_places)
+
+        self.assertAlmostEqual(nxsdata.readY(1)[1700], data1.readY(0)[1700], places=diff_places)
+        self.assertAlmostEqual(nxsdata.readY(1)[2200], data1.readY(0)[2200], places=diff_places)
+
+        # self.assertAlmostEqual(nxsdata.readY(2)[200], data2.readY(0)[200], places=diff_places)
```

Comment:
I think you forgot to uncomment this check? This can be fixed later, there's no shortage of checks!